build/package/.vscodeignore

```diff
@@ -29,3 +29,4 @@
 !src/omega_edit/omega_edit.js
 !src/omega_edit/interface.html
 !src/omega_edit/omega-edit-scala-server*.zip
+!src/language/providers/intellisense/DFDLGeneralFormat.dfdl.xsdf
```

Comment:
Is the `f` at the end of `.dfdl.xsdf` intentional? The file included with Daffodil just has ` .dfdl.xsd` extension. Is this so it doesn't conflict with what Daffodil distributes or something, or is this just a typo? It looks like the actual file does have the `f` as well as a path in `src/language/dfdl.ts`, so if this is typo we are consistent.

Reply:
Yeah it is a typo, but a consistent typo will fix that